src/test/java/core/basesyntax/service/impl/ProcessServiceImplTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ServiceException;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProcessServiceImplTest {
+    private ProcessServiceImpl processService = new ProcessServiceImpl();
+
+    @Before
+    public void beforeEachTest() {
+        Storage.fruitStorage.clear();
+    }
+
+    @Test
+    public void process_return_Ok() {
+        List<String> actual = List.of("type,fruit,quantity",
+                "b,orange,44",
+                "b,banana,30",
+                "r,apple,60",
+                "p,apple,20",
+                "s,orange,75");
+        List<FruitTransaction> expected = List.of(
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "orange", 44),
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 30),
+                new FruitTransaction(FruitTransaction.Operation.RETURN, "apple", 60),
+                new FruitTransaction(FruitTransaction.Operation.PURCHASE, "apple", 20),
+                new FruitTransaction(FruitTransaction.Operation.SUPPLY, "orange", 75));
+        assertEquals(expected, processService.getTransactions(actual));
```

Comment:
looks more clear that way
```suggestion
        List<String> input = List.of("type,fruit,quantity",
                "b,orange,44",
                "b,banana,30",
                "r,apple,60",
                "p,apple,20",
                "s,orange,75");
         List<FruitTransaction> expected = List.of(
                new FruitTransaction(FruitTransaction.Operation.BALANCE, "orange", 44),
                new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 30),
                new FruitTransaction(FruitTransaction.Operation.RETURN, "apple", 60),
                new FruitTransaction(FruitTransaction.Operation.PURCHASE, "apple", 20),
                new FruitTransaction(FruitTransaction.Operation.SUPPLY, "orange", 75));
        List<FruitTransaction> actual = processService.getTransactions(input);
        
        assertEquals(expected, actual);
```